acceptance/acceptance_test.go

```diff
@@ -229,1905 +99,1417 @@ func testWithoutSpecificBuilderRequirement(
-			h.AssertNil(t, err)
-			h.AssertContains(t, output, fmt.Sprintf("Successfully created package '%s'", packageName))
-			return packageName
-		}
-
-		packageBuildpackRemotely := func(absConfigPath string) string {
-			t.Helper()
-			packageName := registryConfig.RepoName("test/package-" + h.RandString(10))
-			output, err := h.RunE(subjectPack("package-buildpack", packageName, "-p", absConfigPath, "--publish"))
-			h.AssertNil(t, err)
-			h.AssertContains(t, output, fmt.Sprintf("Successfully published package '%s'", packageName))
-			return packageName
-		}
-
-		assertImageExistsLocally := func(name string) {
-			t.Helper()
-			_, _, err := dockerCli.ImageInspectWithRaw(context.Background(), name)
-			h.AssertNil(t, err)
-
-		}
-
-		generateAggregatePackageToml := func(buildpackURI, nestedPackageName string) string {
-			t.Helper()
-			packageTomlData := fillTemplate(t,
-				filepath.Join(packFixturesDir, "package_aggregate.toml"),
-				map[string]interface{}{
-					"BuildpackURI": buildpackURI,
-					"PackageName":  nestedPackageName,
-				},
-			)
-			packageTomlFile, err := ioutil.TempFile(tmpDir, "package_aggregate-*.toml")
-			h.AssertNil(t, err)
-			_, err = io.WriteString(packageTomlFile, packageTomlData)
-			h.AssertNil(t, err)
-			h.AssertNil(t, packageTomlFile.Close())
-
-			return packageTomlFile.Name()
-		}
-
 		when("no --format is provided", func() {
 			it("creates the package as image", func() {
 				packageName := "test/package-" + h.RandString(10)
-				output, err := h.RunE(subjectPack("package-buildpack", packageName, "-p", filepath.Join(tmpDir, "package.toml")))
-				h.AssertNil(t, err)
-				h.AssertContains(t, output, fmt.Sprintf("Successfully created package '%s'", packageName))
-				defer h.DockerRmi(dockerCli, packageName)
 
-				assertImageExistsLocally(packageName)
+				output := pack.SuccessfulRunWithOutput(
+					"package-buildpack", packageName,
+					"-p", simplePackageConfigPath,
+				)
+
+				assert.Contains(output, fmt.Sprintf("Successfully created package '%s'", packageName))
+				defer containerManager.RemoveImages(packageName)
+
+				assert.ImageExistsLocally(packageName)
 			})
-		})
 
-		when("--format image", func() {
-			it("creates the package", func() {
+			it("creates the package as a local image", func() {
 				t.Log("package w/ only buildpacks")
-				nestedPackageName := packageBuildpackLocally(filepath.Join(tmpDir, "package.toml"))
-				defer h.DockerRmi(dockerCli, nestedPackageName)
-				assertImageExistsLocally(nestedPackageName)
-
-				t.Log("package w/ buildpacks and packages")
-				aggregatePackageToml := generateAggregatePackageToml("simple-layers-parent-buildpack.tgz", nestedPackageName)
-				packageName := packageBuildpackLocally(aggregatePackageToml)
-				defer h.DockerRmi(dockerCli, packageName)
-				assertImageExistsLocally(packageName)
+				nestedPackageName := newPackageName()
+				assert.PackageCreated(
```

Comment:
I believe `pack.PackageBuildpack` is the "thing" we're testing here, but it's a little hard to glean this from having it occur within the call to `assert.PackageCreated`. I wonder if this could be made clearer.

Reply:
Yes, I think I'll move `PackageCreated` and `PackagePublished` to the OutputAssertionManager anyway, and separate these lines to bring consistency with other tests.